client/lib/create-selector/index.js

```diff
@@ -75,6 +79,10 @@ export default function createSelector( selector, getDependants = DEFAULT_GET_DE
 	let lastDependants;
 
 	return Object.assign( function( state, ...args ) {
+		if ( Array.isArray( getDependants ) ) {
+			getDependants = makeSelectorFromArray( getDependants );
+		}
+
```

Comment:
> Should we move this up to the parent scope?

Yeah, sounds good.

> you could even make the argument that the deferred computation of the `getDependants` selector could be a nice thing

Hm, I hadn't considered it. I honestly placed the call in the child scope out of distraction. If we ever come to need the deferred computation of `getDependants`, we can change it later, but for now I say we move it up.

> We should add some tests, and a doc block on the helper function would be nice.

👍